core/src/test/java/org/testcontainers/utility/MountableFileTest.java

```diff
@@ -0,0 +1,77 @@
+package org.testcontainers.utility;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertFalse;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+
+public class MountableFileTest {
+
+    @Test
+    public void forClasspathResource() throws Exception {
+        final MountableFile mountableFile = MountableFile.forClasspathResource("mappable-resource/test-resource.txt");
+
+        performChecks(mountableFile);
+    }
+
+    @Test
+    public void forClasspathResourceWithAbsolutePath() throws Exception {
+        final MountableFile mountableFile = MountableFile.forClasspathResource("/mappable-resource/test-resource.txt");
+
+        performChecks(mountableFile);
+    }
+
+    @Test
+    public void forClasspathResourceFromJar() throws Exception {
+        final MountableFile mountableFile = MountableFile.forClasspathResource("docker-java.properties");
```

Comment:
Maybe we should use some other file to avoid the situation when they change `docker-java.properties` to something else? Some `META-INF/MANIFEST.MD" IMO is better